scripts/src/l1-resources/cluster/index.ts

```diff
@@ -1,876 +0,0 @@
-// Generated by cdk-import
-import * as cdk from 'aws-cdk-lib';
```

Comment:
Intentional to remove this file?

Reply:
yes, note that it's in script folder, it was probably a mistake when using cdk.sh from script folder instead of root folder